web/templates/popovers/left_sidebar/left_sidebar_condensed_views_popover.hbs

```diff
@@ -1,5 +1,14 @@
 {{! Contents of the condensed nav vdots popup }}
 <ul class="nav nav-list condensed-views-popover-menu">
+    <li class="condensed-views-popover-menu-drafts">
```

Comment:
This class name seems incorrect, it should say `condensed-views-popover-menu-reactions`

Be really careful whenever you copy/paste to actually check that you changed everything.

@alya FYI, this is ![image](https://github.com/zulip/zulip/assets/2746074/e37905a9-45e7-40bf-8d5a-1c994cbe94ca), just making sure you saw that state as a thing to test.

Reply:
Yes, lots of feedback on that above.